### ISSUE TITLE
Cannot find module when importing a ts file in to a js file

### ISSUE THREAD

**Issue by vaibhav-kona** [cypress-io/cypress#8971]
### Current behavior

![Screen Shot 2020-10-26 at 5 42 09 PM](https://user-images.githubusercontent.com/5622964/97170940-95d03580-17b2-11eb-974a-6db8adbb97e4.png)

Using cypress@4.3.0, react@17.0.1, typescript@4.0.3

### Desired behavior

Cypress test should get compiled and run the test assertion. App is already getting compiled without any issues and UI is getting rendered in the browser.

### Test code to reproduce

Repo with test code - https://github.com/vaibhav-kona/test-cypress.git
It has a test at _"test-cypress/cypress/integration/screens/intro.spec.js"_ which is failing with above issue.

### Versions

Cypress - 4.3.0
Browser - Chrome 86
OS - macOS 10.15.7

**Comment by jennifer-shehane** [comment]
Can you double check the repo you linked? Because there's no Cypress code present in that repo. 

I see that you are using an older version of Cypress. Can you please update to the [current version of Cypress](https://on.cypress.io/changelog) and let us know if this is still happening for you. Your issue may have already been fixed. Thanks!

**Comment by vaibhav-kona** [comment]
I pushed the code now for older cypress. I also tested with latest cypress in the "feature/test-with-latest-cypress" branch. 

Now, with latest cypress I am getting a webpack compilation error as in screenshot below when I am trying to import a png file. Again code is getting compiled in the terminal and UI is loaded in browser.

![Screen Shot 2020-10-27 at 12 18 19 PM](https://user-images.githubusercontent.com/5622964/97266734-fca32c80-184e-11eb-9e8d-db229e556d90.png)
